src/diart/sinks.py

```diff
@@ -24,9 +24,12 @@ def __init__(self, path: Union[Path, Text], patch_collar: float = 0.05):
 
     def patch_rttm(self):
         """Stitch same-speaker turns that are close to each other"""
-        annotation = list(load_rttm(self.path).values())[0]
-        with open(self.path, 'w') as file:
-            annotation.support(self.patch_collar).write_rttm(file)
+
+        annotations = list(load_rttm(self.path).values())
+        if len(annotations) > 0:
```

Comment:
```suggestion
        if annotations:
```

This is a more pythonic way of checking for a non-empty list.